bin/setup.sh

```diff
@@ -3,7 +3,7 @@
 # Setup Services
 echo "Setting up Services..."
 ./create-service.sh "ccd_gateway" "false" "ccd_gateway" "OOOOOOOOOOOOOOOO" "[\"http://localhost:3451/oauth2redirect\", \"http://localhost:3002/oauth2/callback\", \"https://localhost:3000/redirectUrl\"]" "[\"caseworker\", \"caseworker-ia\", \"caseworker-ia-legalrep-solicitor\", \"pui-case-manager\"]" "CCD Gateway" "CCD scope manage-user create-user openid profile roles acr search-user authorities"
-
+./create-service.sh "ccd_admin" "false" "ccd_admin" "OOOOOOOOOOOOOOOO" "[\"http://localhost:3451/oauth2redirect\", \"https://localhost:3100/oauth2redirect\"]" "[\"ccd-import\"]" "CCD Gateway" "CCD scope manage-user create-user openid profile roles acr search-user authorities"
```

Comment:
We shouldn't change these urls in master so will need to revert this line.

Reply:
You need this service to run ccd-admin-web locally